frontend/app/package.json

```diff
@@ -18,6 +18,7 @@
     "electron:package": "electron-builder build --config .electron-builder.config.js --publish onTag",
     "electron:serve": "node scripts/serve.js",
     "i18n:report": "vue-i18n-extract report --vueFiles './src/**/*.?(ts|js|vue)' --languageFiles './src/locales/**/*.json'",
+    "link:verify": "tsc src/data/external-links.ts && node scripts/check-external-links.js",
```

Comment:
can't we specify the tsc output to go to some other folder that is already ignored?
or use a tmp folder instead?

Reply:
I wonder if we could use ts-node as we use in the proxy maybe. (and write the checker in ts)

Reply:
let me try to use ts-node